### DISCUSSION TITLE
New translation in Tamil

### DISCUSSION THREAD

**Discussion by VigneshBalamurugan** [sircharlo/meeting-media-manager#1940] [Translations]
### What language do you speak?

I would like to help translate M³ into a language I speak, Tamil (TL/ta).

### Additional context

Hi I am from Tamil Nadu and using this to download our videos for meeting. It would be helpful in some congregation if the app is in Tamil and if possible I would like to contribute for that

**Comment by mtdvlpr** [comment]
@VigneshBalamurugan, thanks for reaching out! Please create an account (using GitHub) at [Crowdin](https://crowdin.com) and go to [this page](https://crowdin.com/project/meeting-media-manager/ta) to request access to our translation project.

**Comment by sircharlo** [comment]
Hi @VigneshBalamurugan ! Thanks for your offer, and let us kow if you need help navigating the translation site 🙂
